### ISSUE TITLE
Incorrect ERROR message in dumping (0.4.7)

### ISSUE THREAD

**Issue by landy2005** [etingof/snmpsim#70]
After upgrade to latest version (0.4.7), dumping any snmp device show (incorrect?) **ERROR OIDs dumped** message:
```
snmprec.py --community=public --use-getbulk --start-oid=1.0 --continue-on-errors=1 --timeout=2 --retries=3 --agent-udpv4-endpoint=X.X.X.X --output-file=/opt/observium-data/full/X.X.X.X
SNMP version 2c, Community name: public
Querying UDP/IPv4 agent at X.X.X.X:161
Agent response timeout: 2.00 secs, retries: 3
Sending initial GETBULK request for 1.0 (stop at <end-of-mib>)....
ERROR OIDs dumped: 0/100
ERROR OIDs dumped: 0/200
...
ERROR OIDs dumped: 0/11900
ERROR Remote SNMP error No SNMP response received before timeout
Retrying with OID 1.3.6.1.3.94.1.11.1.8.16.0.80.235.26.227.70.56.0.0.0.0.0.0.0.0.17792487 (0 retries left)...
ERROR SNMP Engine error: No SNMP response received before timeout
OIDs dumped: 23800, elapsed: 183.18 sec, rate: 64.00 OIDs/sec, errors: 2
# Input file #0, processing records from the beginning till the end
# Records: written 11900, filtered out 0, deduped 0, ignored False, broken 0, variated 0
```

That on any host..

**Comment by etingof** [comment]
Just a quick note that that's the bug in logging level, nothing breaking.

I will push a fix.

Thanks!

**Comment by etingof** [comment]
So this is fixed in the upcoming release 0.4.8. Thanks!
